Okta.Xamarin.Demo.iOS/AppDelegate.cs

```diff
@@ -32,7 +32,7 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
 			//  *** OKTA ***
 			// for demo purposes go to the profile page after sign in and sign out
-			OktaContext oktaContext = OktaPlatform.InitAsync(Window.RootViewController).Result;
+			OktaContext oktaContext = OktaPlatform.InitAsync(Window).Result;
```

Comment:
It isn't clear to me if transpilation to iOS would allow that to function the way one would expect.  I vaguely recall trying this at some point but the behavior didn't seem right so I went back to the current form.